src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionTests.cs

```diff
@@ -6864,6 +6864,30 @@ static void Main()
 }", expectedOutput: "2").VerifyDiagnostics();
         }
 
+        [WorkItem("https://github.com/dotnet/roslyn/issues/70923")]
+        [Fact]
+        public void TestNamedParamsParam_Error()
```

Comment:
Is this a dupe of ```TestNamedParams``` in src\Compilers\CSharp\Test\Semantic\Semantics\NonTrailingNamedArgumentsTests.cs?

Reply:
I do not understand how this test didn't fail when I tried running it earlier. Regardless, I guess I made a molehill out of nothing.